customTests/custom-test-setup.sh

```diff
@@ -95,10 +95,22 @@ else
 			wget https://www.cs.virginia.edu/stream/FTP/Code/stream.c
 		fi
 
+		SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+		if echo $SKU | grep "HX176rs"; then
+			BUILD=ZEN4
+		elif echo $SKU | grep "HB120rs_v3"; then
+			BUILD=ZEN3
+		elif echo $SKU | grep "HB120rs_v2"; then
+			BUILD=ZEN2
+		else
+			#default to zen3 build
+			BUILD=ZEN3
+		fi
+
 		if command -v /opt/AMD/aocc-compiler-4.0.0/bin/clang &> /dev/null; then
-			make CC=/opt/AMD/aocc-compiler-4.0.0/bin/clang EXEC_DIR=$EXE_DIR
+			make $BUILD CC=/opt/AMD/aocc-compiler-4.0.0/bin/clang EXEC_DIR=$EXE_DIR
```

Comment:
Yea thats why we have the else condition here which would require you to add it to the path. I can modify to take in an argument, I'd rather address this in a separate PR since it will affect all HB series and not just HBv2/3